feg/protos/mock_core.proto

```diff
@@ -113,13 +113,28 @@ service MockOCS {
     rpc AssertExpectations(magma.orc8r.Void) returns (GyCreditControlResult) {}
 }
 
+enum FinalUnitAction {
```

Comment:
maybe here if you want, instead of `is_valid` we i mentioned on my first comment, you can add a new field saying  `Nul = -1;` (check if -1 are valid on enum for protobuf).  But it is up to you


In any case, you need to make sure sessiond understand that